src/node/interfaces.cpp

```diff
@@ -487,6 +487,16 @@ class ChainImpl : public Chain
         const CChain& active = Assert(m_node.chainman)->ActiveChain();
         return active.GetLocator();
     }
+    std::optional<CBlockLocator> getLocator(const uint256& block_hash) override
```

Comment:
I think this child method can be private. See #24150 for rationale.

Reply:
> for the sake of consistency I would suggest making all the methods private

So I think it would be better to do this in a follow-up PR since it involves other unrelated methods.